src/index.js

```diff
@@ -229,7 +355,34 @@ class ClientAuthOIDC {
       })
   }
 
-  clearCurrentUser () {
+  /**
+   * Saves given user credentials in storage.
+   *
+   * @param {Object} options
+   */
+  saveCurrentCredentials (options) {
+    this.setCurrentCredentials(options)
+    this.store.setItem(CURRENT_CREDENTIALS, JSON.stringify(options))
+  }
+
+  /**
+   * Initializes user credentials on the client instance.
+   *
+   * @param {Object} options
+   */
+  setCurrentCredentials (options) {
+    this.webId = options.webId
+    this.accessToken = options.accessToken
+    this.idToken = options.idToken
+  }
```

Comment:
IMO, having two sources of truth (storage object vs `this`) for state is going to be difficult to keep track of. At least, just like with `clearCredentials`, there should be only one interface through which to "set" credentials.